apps/src/templates/progress/progressStyles.js

```diff
@@ -65,28 +66,30 @@ const statusStyle = {
  * Given a level object, figure out styling related to its color, border color,
  * and background color
  */
-export const levelProgressStyle = (level, disabled) => {
+export const levelProgressStyle = (levelStatus, levelKind, disabled) => {
   let style = {
-    borderWidth: 2,
+    borderWidth: BUBBLE_BORDER_WIDTH,
+    borderColor: color.lighter_gray,
+    borderStyle: 'solid',
     color: color.charcoal,
     backgroundColor: color.level_not_tried
   };
 
-  if (disabled) {
-    style = {
-      ...style,
-      ...(!disabled && hoverStyle)
-    };
-  } else {
-    if (level.status !== LevelStatus.not_tried) {
-      style.borderColor = color.level_perfect;
-    }
+  if ((disabled && levelStatus !== LevelStatus.submitted) || !levelStatus) {
```

Comment:
i think it would be good to preserve the comment i had here about why we're handling the `submitted` case differently.

Reply:
Added back in- thanks!